code/script.js

```diff
@@ -222,104 +230,205 @@ const generateBoard = () => {
   })
 }
 
+
 // Randomly select a person from the characters array and set as the value of the variable called secret
 const setSecret = () => {
   secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
+   console.log(secret); //To check that a new secret person has been selected 
```

Comment:
Just a quick FYI - this console.log is still working.  So, if the user opens the console when playing they can see the secret person. :)

Reply:
Thanks! I'll fix that now!